doc: add `serviceMonitor` with helm for v1.5.4 (#867)
longhorn/website: content/docs/1.5.4/monitoring/prometheus-and-grafana-setup.md

      - port: manager
    ```

#### Install Longhorn ServiceMonitor with Helm

1. Modify the YAML file `longhorn/chart/values.yaml`.

    ```yaml
    metrics:
      serviceMonitor:

>         # -- Setting that allows the creation of a Prometheus ServiceMonitor resource for Longhorn Manager components.

Reviewer: ```suggestion
        # -- Setting that allows the creation of a [Prometheus Operator](https://prometheus-operator.dev/) ServiceMonitor resource for Longhorn Manager components.
```

Author: Modified with 1.5.4, 1.6.0/1, and 1.7.0. Thanks.

        enabled: true
    ```

1. Create a ServiceMonitor for Longhorn Manager using Helm.

    ```bash
      helm upgrade longhorn longhorn/longhorn --namespace longhorn-system -f values.yaml
    ```

> Longhorn ServiceMonitor is included in the Prometheus custom resource. This setup allows the Prometheus server to discover all Longhorn Manager pods and their respective endpoints.

Reviewer: ```suggestion
Longhorn ServiceMonitor is a [Prometheus Operator](https://prometheus-operator.dev/) custom resource. This setup allows the Prometheus server to discover all Longhorn Manager pods and their respective endpoints.
```

Author: Modified with 1.5.4, 1.6.0/1, and 1.7.0. Thanks.
